apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -167,7 +167,7 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
     const orgType: OrganizationType = useSelector((state: AppState) =>
         state?.organization?.organizationType);
     const isFAPIAppCreationEnabled: boolean = useSelector((state: AppState) =>
-        state.config.ui.features?.fapiApplicationCreation?.enabled);
+        !state.config.ui.features?.applications?.disabledFeatures?.includes("fapiApplicationCreation"));
```

Comment:
`fapiApplicationCreation` lets take this from a constant.
Check if there's already a map to store the feature ids.

REF: https://github.com/wso2/identity-apps/blob/c96eb8b986c93451340dc1f2f3bd86201f7f2c81/apps/console/src/features/groups/constants/group-constants.ts#L40